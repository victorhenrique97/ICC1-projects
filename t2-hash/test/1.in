13
9
44
45
49
70
27
73
92
97
95
