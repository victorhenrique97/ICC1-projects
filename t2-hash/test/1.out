0:
1: 27 92
2:
3:
4: 95
5: 44 70
6: 45 97
7:
8: 73
9:
10: 49
11:
12:
